.github/workflows/push_docker_image.yml

```diff
@@ -73,8 +73,8 @@ jobs:
       - name: push git tagged version
         run: |
           docker pull $DOCKER_REPO:git-${{ github.sha }}-amd64
-          if [[ -z ${{ github.events.inputs.imageTag }} ]]; then
-            export IMAGE_TAG=${{ github.events.inputs.imageTag }}
+          if [[ -z ${{ github.event.inputs.imageTag }} ]]; then
+            export IMAGE_TAG=${{ github.event.inputs.imageTag }}
           else
```

Comment:
Does it set `IMAGE_TAG` when the `imageTag == ''`?

```
$ if [[ -z "" ]]; then echo 1; fi 
1
$ if [[ -z "sometag" ]]; then echo 1; fi
# no output
```

Reply:
oooh, i confused, thanks